File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/lock/SignedMessagePublicKeyPayloadResponse.java
package com.lattis.ellipse.data.network.model.response.lock;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 4/27/17.
 */

public class SignedMessagePublicKeyPayloadResponse {

    @SerializedName("signed_message")
    private String signed_message;

    @SerializedName("public_key")
    private String public_key;

    public String getSigned_message() {
        return signed_message;
    }

    public void setSigned_message(String signed_message) {
        this.signed_message = signed_message;
    }

    public String getPublic_key() {
        return public_key;
    }

    public void setPublic_key(String public_key) {
        this.public_key = public_key;
    }





}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/lock/SignedMessagePublicKeyPayloadResponse.kt
package com.lattis.data.entity.response.lock

import com.google.gson.annotations.SerializedName

class SignedMessagePublicKeyPayloadResponse {
    @SerializedName("signed_message")
    var signed_message: String? = null
    @SerializedName("public_key")
    var public_key: String? = null

}